lib/validate.js

```diff
@@ -122,6 +122,13 @@ const validateCategory = (
     const isSchemaProp = schemaProps.has(prop);
     const isObjectProp = objectProps.has(prop);
     if (isObjectProp && !isSchemaProp) {
+      if (
```

Comment:
```suggestion
      if (schema[Symbol.for('other')])
```

Should work just fine.